solutions/instances/variables.tf

```diff
@@ -265,7 +265,7 @@ variable "existing_kms_instance_crn" {
 variable "existing_cos_kms_key_crn" {
   type        = string
   default     = null
-  description = "The CRN of an existing KMS key to be used to encrypt both the COS bucket i.e 'log-archive-bucket' and 'at-target-cos-bucket'. If not supplied, a new key ring and key will be created in the provided KMS instance."
+  description = "(OPTIONAL) The CRN of an existing KMS key to be used to encrypt both the COS bucket i.e 'log-archive-bucket' and 'at-target-cos-bucket'. If not supplied, a new key ring and key will be created in the provided KMS instance."
```

Comment:
This is not the description I put in https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/42#discussion_r1570627838